code/src/components/survey-components/Secondquestion.js

```diff
@@ -0,0 +1,66 @@
+import React, {useState} from 'react'
+
+const ageGroups = ['0-18','19-30','30+']
+
+const Secondquestion = (props) => {
+  const [showAgegroup, setShowAgegroup] = useState(false)
+
+  const handlePersonnumber = (e) => {
+    props.setPersonNumber(e.target.value)
+    props.setError('')
+    setShowAgegroup(true)
+    }
+
+    const handleAgegroup = (e) => {
+      props.setAgeGroup(e.target.value)
+      props.setError('')
+    }
+
+    const ageGroupsInfo = () => {
+      if(props.personNumber === ''){
+        setShowAgegroup(false)
+      }
+      return (
+        <section className="question-section">
+        <label htmlFor='name' tabIndex={0}>Age group:</label>
+        {ageGroups.map((agegroups,key) => 
+        <label htmlFor="ageGroup" key={agegroups} className='agegroup-label' tabIndex={0}>
+        <input
+            className='agegroup-input' 
+            id="ageGroup"
+            type = 'radio' 
+            value={agegroups}
+            onChange={handleAgegroup} 
+            checked={props.ageGroup === agegroups}
+            tabIndex={0}
+        />
```

Comment:
I could be wrong but if you move the htmlFor='ageGroup' to inside <input /> it will also be possible to click on the text next to the button and the button will be ticked in, instead of only the radio button. Let me know if I exaplained it wrong here, we could talk about it more :)